src/sherpa_ai/agents/mathematician.py

```diff
@@ -0,0 +1,169 @@
+import os
```

Comment:
In general, I think the mathematician is not too different from other agents except for the action of solving a math problem. I think we should implement that as an action so it can be configured into an existing agent to make that agent a _mathematician_.

Reply:
i add a new action answer_arithmetic.py
Do I need to remove the math agent? @20001LastOrder

Reply:
Yes, I think the agent can be removed. And later we will figure out a configuration for the mathematician,

Reply:
fixed